client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -158,7 +159,12 @@ PluginAutoUpdateToggle.defaultProps = {
 };
 
 export default connect(
-	null,
+	( state, { disabled, plugin, site } ) => ( {
+		disabled: disabled || (
+			get( site, 'options.is_automated_transfer', false ) &&
```

Comment:
Let's use a selector here `isSiteAutomatedTransfer` from `state/selectors/is-site-automated-transfer`